lib/widget.dart

```diff
@@ -350,42 +350,57 @@ class _StoriesList extends StatelessWidget {
 
     stories.forEach((story) {
       children.add(
-        Text(
-          story.name,
-          style: TextStyle(fontWeight: FontWeight.bold, fontSize: 20),
-        ),
-      );
-      children.add(SizedBox(height: 10));
-
-      story.chapters.forEach((chapter) {
-        children.add(
-          GestureDetector(
-            behavior: HitTestBehavior.opaque,
-            child: _Link(
-              label: "  ${chapter.name}",
-              textAlign: TextAlign.left,
-              padding: EdgeInsets.zero,
-              height: 20,
-              textStyle: TextStyle(
-                fontWeight: chapter.id == selectedChapter.id
-                    ? FontWeight.bold
-                    : FontWeight.normal,
-              ),
+        ExpansionTile(
+          title: Text(
+            story.name,
+            style: TextStyle(
+              fontWeight: FontWeight.bold,
+              fontSize: 20,
+              color: Colors.black,
             ),
-            onTap: () {
-              onSelectChapter(chapter);
-            },
           ),
-        );
-      });
+          initiallyExpanded: true,
```

Comment:
**question**: Could you convert this into a stateful widget and maintain the expanded state per story defaulting to all collapsed? That way you don't have a huge list when the app opens and when you go into a preview and back to the story list, it maintains the expanded state per story.

Reply:
We may want to update to the latest version of dashbook, since there have been a lot of positive changes. @russell-sf

Reply:
@russell-sf i took a look at the effort for persisting state.  w/o bringing in another lib this will be a bigger fix because there isn’t currently any routing. The pages are essentially being conditionally rendered in a stack based on an enum value which means they are rebuilt every time.

Reply:
**question:** how about defaulting to collapsed? I think that would be a better experience until we could persist state.

Reply:
@jeffs-sf changed to collapsed by default.